### PR TITLE
Pushable deployment

### PR DESCRIPTION
**Pull request by SamMaier**
basically all the stuff needed for deployment that can reasonably be put in master

### CODE REVIEW

**Comment by samiskin** [comment]
Does this still work on local?  Wouldn't we need to set `PORT`?

File: src/server/server.ts [hunk 1]
```diff
@@ -11,8 +13,8 @@ app.get('/', function (req: express.Request, res: express.Response) {
   res.sendFile(path.resolve(__dirname, '../index.html'));
 });
 
-server.listen(3000, function () {
-  console.log('listening on *:3000');
+server.listen(process.env.PORT || 3000, function () {
+  console.log('Listening on port ', process.env.PORT);
```
Comment (samiskin) [inline src/server/server.ts]: Would need the `|| 3000)` here too right?

Comment (SamMaier) [inline src/server/server.ts]: Done

**Comment by samiskin** [comment]
👍

**Pull request merged**